Formula/idris2.rb

```diff
@@ -0,0 +1,31 @@
+class Idris2 < Formula
+  desc "Pure functional programming language with dependent types"
+  homepage "https://www.idris-lang.org/"
+  url "https://github.com/idris-lang/Idris2/archive/v0.2.0.tar.gz"
+  sha256 "cc7c48d3fbaa55e536890c9eb4701755081d2b5df9a65f0ecc3e00ee5ccf0a0f"
+
+  depends_on "coreutils" => :build
+  depends_on "chezscheme"
```

Comment:
Hi, @vladimyr shouldn't "chezscheme" be only a build time dependency, or is it needed for runtime code generation too? 
Thank you for your efforts, I'm eager to see this PR get merged :)

Reply:
It is used as Idris 2 compiler backend which makes it required in _runtime_. You are welcome, I'm doing it in appreciation for all the great things Idris folks created and thought us 😉